README.md

```diff
@@ -17,14 +17,17 @@ designed as libraries (for programmatic use) with minimal command line interface
 
 ## Installation
 
-After cloning the Git repository, on the command line do execute
+To get setup with the OSS Review Toolkit, simply:
+
+1. Ensure the Java JDK (not the JRE as you need the `javac` compiler) is installed and the JAVA_HOME environment variable set.
```

Comment:
While at it, we should probably say that we target Java 8. Java 9 *should* work now that we use Gradle 4.4 (see [this issue](https://github.com/gradle/gradle/issues/3317)), but I haven't verified it.